README.md

```diff
@@ -487,45 +466,32 @@ The Identity component is implemented in the [adex-protocol-eth repository](http
 
 #### Pre-approved tokens
 
-While OUTPACE can work with any Ethereum token that implements the ERC20 standard, not all of them are suitable for using as campaign deposit. Some tokens have fatal bugs, others allow arbitrary minting, and some are simply not liquid enough.
-
-This is why we came up with a set of pre-approved tokens. For now, we've decided on USDC and ADX, but we can easily allow more.
-
-It's important to note that **this is not enforced on a blockchain/smart contract level**, but it's merely a UI limitation. If you feel that a certain token should be added, you can submit a PR to [`adex-platform`](https://github.com/AdExNetwork/adex-platform).
-
+Although OUTPACE has the capability to integrate with any Ethereum token adhering to the ERC20 standard, not all tokens are suitable for campaign deposits due to factors such as fatal bugs, arbitrary minting, or insufficient liquidity. While the smart account associated with each AdEx profile supports various networks and tokens, the AdEx platform specifically operates on the Ethereum and Polygon networks. Depending on the chosen network, campaign deposits can be made using USDC, USDT, DAI, and ADX. Launching a campaign with the ADX token triggers an automatic reduction in the required transaction fee for each campaign from 7% to 4%.
```

Comment:
link "made using USDC, USDT" to this: https://snapshot.org/#/adex.eth/proposal/0x03be563ddaf06ba0563baaa212833c33b8f85e895a3edbb563fe2ffd1e044244